stackslib/src/chainstate/stacks/boot/signers_voting_tests.rs

```diff
@@ -359,15 +368,25 @@ fn vote_for_aggregate_public_key_in_last_block() {
         })
     );
 
+    // third vote should succeed even though it is on an old round
+    let tx3 = &receipts[receipts.len() - 1];
+    assert_eq!(
+        tx3.result,
+        Value::Response(ResponseData {
+            committed: true,
+            data: Box::new(Value::Bool(true))
+        })
```

Comment:
```suggestion
        Value::okay_true()
```

Reply:
Also it's not this PR's code but `Value::error(Value::UInt(10002)).unwrap()` for errors